src/datasets/utils/resources/languages.json

```diff
@@ -139,6 +140,7 @@
     "byn": "Blin; Bilin",
     "bzd": "Bribri",
     "ca": "Catalan",
+    "cat": "Catalan; Valencian",
```

Comment:
The same as above: for "Catalan; Valencian" we use `ca` instead. I would remove `cat` from README and from here.
```suggestion
```